### PR TITLE
Update package xmls

### PR DESCRIPTION
**Pull request by bmagyar**
This is a conversion to package.xml format 2, some cleanup and updating the maintainers.

### CODE REVIEW

File: combined_robot_hw/package.xml [hunk 1]
```diff
@@ -12,16 +14,9 @@
   <url type="repository">https://github.com/ros-controls/ros_control</url>
 
   <author>Toni Oliver</author>
```
Comment (efernandez) [inline combined_robot_hw/package.xml]: pedantic: `<author  email="toni@shadowrobot.com">Toni Oliver</author>`

File: combined_robot_hw_tests/package.xml [hunk 2]
```diff
@@ -12,20 +14,15 @@
   <url type="repository">https://github.com/ros-controls/ros_control</url>
 
   <author>Toni Oliver</author>
```
Comment (efernandez) [inline combined_robot_hw_tests/package.xml]: pedantic: `<author  email="toni@shadowrobot.com">Toni Oliver</author>`

File: combined_robot_hw_tests/package.xml [hunk 3]
```diff
@@ -12,20 +14,15 @@
...
-  <build_depend>rostest</build_depend>
-  <run_depend>combined_robot_hw</run_depend>
-  <run_depend>controller_manager</run_depend>
-  <run_depend>controller_manager_tests</run_depend>
-  <run_depend>hardware_interface</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>rostest</run_depend>
+  <depend>combined_robot_hw</depend>
+  <depend>controller_manager</depend>
+  <depend>controller_manager_tests</depend>
```
Comment (efernandez) [inline combined_robot_hw_tests/package.xml:21]: Not sure, but I guess this one is only used for tests, so it should be `<test_depend>`

File: controller_interface/package.xml [hunk 4]
```diff
@@ -14,14 +16,7 @@
...
-  <build_depend>hardware_interface</build_depend> 
-  <build_depend>pluginlib</build_depend> 
-  <run_depend>roscpp</run_depend> 
-  <run_depend>hardware_interface</run_depend> 
-  <run_depend>pluginlib</run_depend> 
-
-  <export>
-    <cpp cflags="-I${prefix}/include"/>
-  </export>
+  <depend>roscpp</depend>
```
Comment (efernandez) [inline controller_interface/package.xml]: pedantic: alphabetize (here and others). In `vim` you only need to do `:!sort -u` (`-u` is for unique)

Comment (efernandez) [reply controller_interface/package.xml]: Given how big is the PR, I think this could be done on another PR if you prefer.

**Review by efernandez — APPROVED** [review]
Only some minor/style comments, but looks good to merge regardless.

**Comment by efernandez** [comment]
Thanks for taking the time to upgrade to package.xml format 2. Now things looks much simpler.

File: controller_interface/package.xml (hunk 4 above)
Comment (bmagyar) [reply controller_interface/package.xml]: Thanks for the tip, I'll try to memorize it!

File: combined_robot_hw_tests/package.xml (hunk 3 above)
Comment (bmagyar) [reply combined_robot_hw_tests/package.xml:21]: This package has some binaries that are compiled in normal mode, we may want to revise that  part though.

**Comment by bmagyar** [comment]
Going to merge once build has completed...

**Pull request merged**